template/circle.yml

```diff
@@ -14,6 +17,8 @@ dependencies:
     - pip install xml2rfc
     - if head -1 -q *.md | grep '^\-\-\-' >/dev/null 2>&1; then gem install --no-doc kramdown-rfc2629; fi
     - if head -1 -q *.md | grep '^%%%' >/dev/null 2>&1; then go get "$mmark_src" && go build "$mmark_src"; fi
+  cache_directories:
+    - "/opt/circleci/.rvm/gems"
```

Comment:
Do you think we should set KRAMDOWN_REFCACHEDIR to ~/.cache and cache that as well?